courses/free-courses-bn.md

```diff
@@ -119,6 +119,7 @@ Online School
 
 ### HTML and CSS
 
+* [CSS3 Essential Training (Bangla)](https://www.youtube.com/playlist?list=PLSNRR4BKcowA9IsN4F5utx7OlWUdN0RZV) - Procoder BD
```

Comment:
could you alphabetize the "HTML and CSS" list? `CSS3` should come after `CSS ` Thanks